src/bundle/Resources/config/views.yml

```diff
@@ -13,5 +13,5 @@ system:
             versions_tab:
                 default:
                     template: "EzSystemsHybridPlatformUiBundle:content/tabs:versions.html.twig"
-                    controller: "EzSystemsHybridPlatformUiBundle:ContentView:versionsTab"
+                    controller: "EzSystems\\HybridPlatformUiBundle\\Controller\\ContentViewController:versionsTabAction"
```

Comment:
Valid argument, but I still think that we should go for single quotes. We could make it consistent by going single quotes everywhere. But then I'd prefer a distinct commit, if we can keep the PRs simple.